README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+Hey there!  My name is Kristina Wald.  Here are some quick facts about me:
+
+# Age
+25
+
+# Where I'm from
```

Comment:
Good practice is to leave a line break between headers and blocks of text (or really between two different types of markdown content). Avoids formatting issues later on